src/components/UsagePieChart.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { bytesToDisplayValue } from './../utils/byteUtils';
+
+
+interface Props {
+  available: number;
+  used: number;
+}
+
+const UsagePieChart = (props:Props) => {
+  const {available, used} = props;
+  const percentageUsed = (used/available) * 100;
+  const percentageNotUsed = 100 - percentageUsed;
+  return (
+      <div
+      style={{
+        position: "relative",
+        alignItems: "flex-start",
+        display: "inline-flex",
+        flexDirection: "column",
+      }}
+      >
+        <div 
+          style={{
+            position: "relative",
+            alignItems: "flex-start",
+            display: "inline-flex",
+            flexDirection: "column",
+          }}
+        >
+          <svg
+            height={138}
+            width={138}
+            viewBox={"-18 -18 36 36"}
```

Comment:
I think this svg for the donut chart can preferably be made into a reusable component (in ts file like in the 3di so it can also receive props). But since the donut chart is being used only here so it's up to you whether you think it's useful to make it reusable.

Reply:
Good idea to make it more reusable.
I made this ticket:
https://github.com/nens/lizard-management-client/issues/936